### PR TITLE
add missing Automake dependency to PhyML 3.3.20200621

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by sassy-crick** [comment]
During the installation inside my Singularity container the original EasyConfig crashed, as I don't have `Automake` installed in the container. That is now the fix for it.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/2ac0625aca96c09babb3b4b5d59e283f for a full test report.

File: easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb [hunk 1]
```diff
@@ -22,6 +22,7 @@ checksums = ['a8243923ee08c74cab609a4b086ade66c6156fc2b24450e2a500108dc644c867']
 builddependencies = [
     ('Autoconf', '2.69'),
     ('pkg-config', '0.29.2'),
+    ('Automake', '1.16.2'),
```
Comment (bedroge) [inline easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb:25]: As we need both `Autoconf` and `Automake`, it's better to switch to `Autotools` (which contains both):
```suggestion
    ('Autotools', '20200321'),
    ('pkg-config', '0.29.2'),
```

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb:25]: Also, please make the same change to `PhyML-3.3.20190321-foss-2018b.eb`, to keep things consistent.

Comment (sassy-crick) [reply easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb]: Done. I have done the replacement as suggested, which makes more sense, and I have updated the `PhyML-3.3.20190321-foss-2018b.eb` file as requested well.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb (hunk 1 above)
Comment (bedroge) [reply easybuild/easyconfigs/p/PhyML/PhyML-3.3.20200621-foss-2020b.eb]: Great, thanks @sassy-crick!

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/e379a325d936b3516a4fcc02a4a9d177 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm!

A simply sanity check command doesn't seem to work here, unfortunately, as `phyml --help/--version` returns 1...

**Comment by bedroge** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**